timbuctoo-instancev4/src/test/java/nl/knaw/huygens/timbuctoo/v5/dataset/DataSetFactoryTest.java

```diff
@@ -104,16 +108,16 @@ public void dataSetExistsReturnsFalseIfTheUserDoesNotOwnADataSetWithTheDataSetId
 
   @Test
   public void dataSetExistsReturnsTrueIfTheUserOwnsADataSetWithTheDataSetId() throws DataStoreCreationException {
-    dataSetFactory.createImportManager("ownerId", "dataSetId");
+    getImportManager("ownerId", "dataSetId");
 
     boolean dataSetExists = dataSetFactory.dataSetExists("ownerId", "dataSetId");
 
     assertThat(dataSetExists, is(true));
   }
 
   @Test
-  public void removeDataSetRemovesTheDataSetFromDisk() throws Exception {
-    dataSetFactory.createImportManager("user", "dataSet");
+  public void deleteDataSetRemovesTheDataSetFromDisk() throws Exception {
+    getImportManager("user", "dataSet");
```

Comment:
For me it is more logical if the test starts with the name of the method it is testing.